testutils/lint: TestLint failed

testutils/lint.TestLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/BUILDTYPE_ID-not-found-in-env/7417139?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/BUILDTYPE_ID-not-found-in-env/7417139?buildTab=artifacts#/) on release-22.2.0 @ [234c9295cc02150f919cfa96b09ee2fa07b68ace](https://github.com/cockroachdb/cockroach/commits/234c9295cc02150f919cfa96b09ee2fa07b68ace):


```
=== RUN   TestLint
--- FAIL: TestLint (1290.26s)
=== RUN   TestLint/TestStaticCheck
    lint_test.go:1784: 
        pkg/sql/schemachanger/scbuild/internal/scbuildstmt/helpers.go:139:3: this linter directive didn't match anything; should it be removed? (staticcheck)
    lint_test.go:1784: 
        pkg/sql/schemachanger/scbuild/internal/scbuildstmt/helpers.go:142:3: this linter directive didn't match anything; should it be removed? (staticcheck)
    --- FAIL: TestLint/TestStaticCheck (1039.84s)
```
<p>Parameters: <code>GOFLAGS=-json</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91611 testutils/lint: TestLint failed [C-test-failure O-robot branch-release-22.2]
</p>
</details>
/cc @cockroachdb/test-eng
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-21351

--- a/pkg/sql/schemachanger/scbuild/internal/scbuildstmt/helpers.go
+++ b/pkg/sql/schemachanger/scbuild/internal/scbuildstmt/helpers.go
@@ -136,10 +136,8 @@ func errMsgPrefix(b BuildCtx, id catid.DescID) string {
 func dropElementWhenDroppingDescriptor(b BuildCtx, e scpb.Element) {
 	switch t := e.(type) {
 	case *scpb.ColumnType:
-		//lint:ignore SA1019 IsRelationBeingDropped is deprecated
 		t.IsRelationBeingDropped = true
 	case *scpb.SecondaryIndexPartial:
-		//lint:ignore SA1019 IsRelationBeingDropped is deprecated
 		t.IsRelationBeingDropped = true
 	}
 	b.Drop(e)

